assets/src/edit-story/elements/text/output.js

```diff
@@ -139,14 +129,18 @@ export function TextOutputWithUnits({
   };
 
   if (backgroundTextMode === BACKGROUND_TEXT_MODE.HIGHLIGHT) {
+    const foregroundContent = getValidHTML(content);
+    const backgroundContent = getValidHTML(content, (node) =>
+      removeInlineStyle(node, 'color')
```

Comment:
I'm a bit lost why we need a separate PR. Also commented on the same in https://github.com/google/web-stories-wp/pull/1323#discussion_r419765427. We already have DraftJS utils to do this. But not a blocker to merge.

Reply:
/s/PR/util

Reply:
Replied above.